### PR TITLE
correcting errors found by students

### PR DESCRIPTION
**Pull request by MSilva95**

### CODE REVIEW

File: subjects/nrune/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### Instructions
 
-Write a function that returns the nth `rune` of a `string`. If not possible, it returns `0`.
+Write a function that returns the nth `rune` of a `string`. If it's not possible it doesn't return anything.
```
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/nrune/README.md:5]: For this one i thought from the point of visual representation, visually it doesn't return nothing but i guess you are correct. i will leave it like it was it looks better

File: subjects/printstr/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ And its output :
 
 ```console
 $ go run . | cat -e
-Hello World!%
+Hello World!$
```
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/printstr/README.md:33]: Yes, this one i didn't even check, just saw the mistake % and changed it, my bad i will remove

**Review by davhojt — APPROVED** [review]

**Comment by davhojt** [comment]
@rcatini At the moment, the prompt and `cat -e`'s new-line character are both represented with a `$`. Some commentary on that is here: https://github.com/01-edu/public/issues/1304#issuecomment-1172544466

At the moment, we're considering some ways to make this more clear.
Do you have a proposal?

**Comment by rcatini** [comment]
My proposal for the prompt would be something completely nonstandard, like a right arrow  `→` or `bim>`.

But to make the subject clearer, there is another way: just print a newline in the proposed test program. It's true that it would clutter a bit the test, but only a handful of subjects are affected by this type of partial line printing.

**Pull request merged**